server/models/TrustModel.js

```diff
@@ -0,0 +1,52 @@
+const expect = require('expect-runtime');
+const knex = require('../../server/database/knex');
+const HttpError = require("./HttpError");
+const EntityModel = require("./EntityModel");
+
+class TrustModel{
+  async get(){
+    //const trust_relationship_instance = new trust_relationship(1);
+    const list = await knex.select()
+      .table("trust_relationship");
+    return list;
+  }
+
+  async request(requestType, walletName){
+    expect(requestType, () => new HttpError(`The trust request type muse be one of ${Object.keys(TrustModel.ENTITY_TRUST_REQUEST_TYPE).join(',')}`, 400))
```

Comment:
I build this tool, to verify the input, and by passing in the exception factory, so it can throw an error when failed to verify the data.
I think in this way, we can handle data checks and error throwing pretty easily.